### PR TITLE
fixes bug 1357833 - Reflect jobs that don't run in /crontabber-state

### PR DESCRIPTION
**Pull request by peterbe**
I didn't want to spend too much time on this. But here's something that works. 
screen shot 2017-04-19 at 2 49 20 pm

screen shot 2017-04-19 at 2 50 05 pm

### CODE REVIEW

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/crontabber_state.js [hunk 1]
```diff
@@ -205,6 +220,9 @@ d3.json("/api/CrontabberState/", function(data) {
     var tr = tbody.selectAll("tr")
         .data(nodes)
       .enter().append("tr")
```
Comment (adngdb) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/crontabber_state.js]: nit: correct indentation here.

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/crontabber_state.js [hunk 2]
```diff
@@ -289,6 +307,60 @@ d3.json("/api/CrontabberState/", function(data) {
...
+    tableFields = [
+        'name',
+        'last_run',
+    ];
+    var anyObsolete = false;
+
+    table.classed('data-table tablesorter', true);
+    thead.append("tr").selectAll("th")
+        .data(tableFields)
+      .enter()
```
Comment (adngdb) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/crontabber_state.js]: nit: correct indentation here.

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/crontabber_state.js [hunk 3]
```diff
@@ -289,6 +307,60 @@ d3.json("/api/CrontabberState/", function(data) {
...
+        .append("th")
+
```
Comment (adngdb) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/crontabber_state.js]: Why a new line here?

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/crontabber_state.js [hunk 4]
```diff
@@ -289,6 +307,60 @@ d3.json("/api/CrontabberState/", function(data) {
...
+        .text(function(d, i) {
+            field = tableFields[i];
+            if (field === 'last_run') {
+                return moment(d).fromNow(false);
+            }
+            return d;
+        });
+
+    if (anyObsolete) {
+        // if there are any ongoing rows, only they display the ongoing panel
```
Comment (adngdb) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/crontabber_state.js]: s/ongoing/obsolete

**Comment by peterbe** [comment]
@adngdb I couldn't see my own nits for all the eslint errors so I cleaned up the whole file. Still seems to work just fine. Let's merge when green again.

**Pull request merged**